### ISSUE TITLE
HighContrast icon theme

### ISSUE THREAD

**Issue by Obsidien** [thinkle/gourmet#789]
GNOME[1] and Gtk (integrated in 3.15.3) support a second icons’ theme than the usual, called HighContrast, designed for accessibility: black and white (and sometimes a little grey), big lines, simple icons’ design. It would be great that Gourmet offers such an icon.

[1] https://wiki.gnome.org/Initiatives/GnomeGoals/HighContrastAppIcons

**Comment by ockham** [comment]
I like the idea, but I'm not very good with Inkscape :-( Any volunteers?

**Comment by Obsidien** [comment]
A technical note: in fact, HighContrast icons are not anymore the prefered way to do, but the bug is always here. The goal is now to offer a *symbolic icon* in the hicolor theme. That’s a svg icons with some caracteristics that permit to change its rendering depending on the context ([read infos on Askubuntu](http://askubuntu.com/questions/148955/why-do-the-gnome-symbolic-icons-appear-darker-in-a-running-application)). Maybe the cookie shape would be distinctive enough? possible.
